scipy/stats/_qmc.py

```diff
@@ -1543,19 +1550,16 @@ def random(self, n: _int = 1) -> np.ndarray:
             return transf_samples[:, : self.d]
 
 
-class MultivariateNormalQMC(QMCEngine):
-    r"""QMC sampling from a multivariate Normal :math:`N(\mu, \Sigma)`.
+class MultinomialQMC(QMCEngine):
+    r"""QMC sampling from a multinomial distribution.
 
     Parameters
     ----------
-    mean: array_like (d,)
-        The mean vector.
-    cov: array_like (d, d)
-        The covariance matrix.
-    inv_transform: bool, optional
-        If True, use inverse transform instead of Box-Muller. Default is True.
+    pvals: Iterable[float]
+        float vector of probabilities of size `k`, where `k` is the number of
+        categories. Elements must be non-negative and sum to 1.
     engine: QMCEngine, optional
-        Quasi-Monte Carlo engine sampler. If None, Sobol' is used.
+        Quasi-Monte Carlo engine sampler. If None, Sobol is used.
```

Comment:
```suggestion
        Quasi-Monte Carlo engine sampler. If None, Sobol' is used.
```

Reply:
What is `Sobol'` ?

Reply:
Oh sorry I just realized you wanted to refer to the object so it should be ``Sobol``